apps/web/modules/surveys/components/QuestionFormInput/components/MultiLangWrapper.tsx

```
@@ -0,0 +1,104 @@
+                selectedLanguageCode={usedLanguageCode}
+                surveyLanguages={localSurvey.languages}
+                setSelectedLanguageCode={setSelectedLanguageCode}
+                locale={locale}
+              />
+            ) : null,
+        })}
+      </div>
+
+      {enabledLanguages.length > 1 && (
+        <>
+          {usedLanguageCode !== "default" && value && typeof value["default"] !== undefined && (
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)